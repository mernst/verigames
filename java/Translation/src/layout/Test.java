package layout;

import level.Board;
import level.Level;
import level.World;
import level.WorldXMLPrinter;
import level.levelWorldCreation.IntersectionLevel;
import level.levelWorldCreation.LevelWorld;

public class Test
{
   public static void main (String[] args)
   {
      Layout.main(args);
   }
}

class Layout
{
   public static void main(String[] args)
   {
      World w = LevelWorld.getWorld();
      new WorldLayout().layout(w);
      new WorldXMLPrinter().print(w, System.out, null);
   }
}

class Dot
{
   public static void main(String[] args)
   {
      Board b;
      {
         Level l = IntersectionLevel.makeLevel();
         l.deactivate();
         b = l.getBoard("Intersection.padToLength");
      }
     
      
      DotPrinter p = new DotPrinter();
      
      p.print(b, System.out, null);
   }
}
