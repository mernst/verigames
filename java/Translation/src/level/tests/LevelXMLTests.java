package level.tests;

import org.junit.Test;

// class TestClass
// {
// String s;
//
// public TestClass()
// {
// s = null;
// }
//
// public void method()
// {
// s = new String("asdf");
// }
// }

public class LevelXMLTests
{
   
   /**
    * Generates the XML for the above TestClass
    */
   @Test public void test1()
   {
      
   }
}
