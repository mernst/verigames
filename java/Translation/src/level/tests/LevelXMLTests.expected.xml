
<!DOCTYPE level SYSTEM "level.dtd">
